app/components/DbEntry/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+*
+* DbEntry
+*
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import { getActive } from 'themes';
+import PropTypes from 'prop-types';
+
+class DbEntry extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.minmax = this.minmax.bind(this);
+    this.state = { toggle: false };
+  }
+  minmax(evt) {
+    evt.preventDefault();
+    // this.setState({ toggle: !this.state.toggle });
+  }
+  fill() {
+
+  }
+  mochEntries = Object.entries(this.props.moch);
+  mochComponents = this.mochEntries.map((mochEntry) => <p style={{ display: 'inline-block', padding: '10px' }} > {mochEntry[0]}: {mochEntry[1]}</p>);
+  render() {
+    const theme = getActive();
+    const Entry = styled.div`
+      border: 1px solid black;
+      margin:auto;
+      width:100%;
+      height:auto;
+      background-color: ${theme.get('tertiary')};
+      color: $(theme.get('secondary_text'));
+      position:relative;
+      > h3 {
+        position: absolute;
+        top: 0px;
+        margin: 0px;
+      }
+      &.isActive {
+        background-color: lightgreen;
+      }
+    `;
+    // const open = {
```

Comment:
Same as above vestigial code should either be removed or commented to explain what it is there for.